index.js

```diff
@@ -1,8 +1,18 @@
 'use strict';
 const pReduce = require('p-reduce');
+const is = require('@sindresorhus/is');
 
 module.exports = iterable => {
 	const ret = [];
+
+	for (const task of iterable) {
+		const type = is(task);
+
+		if (type !== 'Function') {
```

Comment:
We don't really need all of `is` for this. `typeof task !== 'function'` should be good enough.

Reply:
Yeah, but a common mistake is to supply a `Promise` instead of a `function`. If we just do `typeof` it'll log it as `object` which isn't super clear.

Reply:
I see. Ok, leave it. We'll switch to `ow` soon enough anyway.